asset-transfer-basic/chaincode-external/assetTransfer.go

```diff
@@ -0,0 +1,219 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/hyperledger/fabric-chaincode-go/shim"
+	"github.com/hyperledger/fabric-contract-api-go/contractapi"
+)
+
+type ServerConfig struct {
+	CCID    string
+	Address string
+}
+
+// SmartContract provides functions for managing a car
+type SmartContract struct {
```

Comment:
I'm not sure, I actually need to try this out, but does `SmartContract` actually have to be exported